CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Fixes for `danger.import_dangerfile "org/repo"` - orta re:#487
 * Remove a variable assignment in comments_helper.rb - JuanitoFatas
+* Prints updated danger message when installed danger is outdated - JuanitoFatas
```

Comment:
can you move this up into the master section please ( https://github.com/dblock/danger-changelog/issues/9 )